x-pack/plugins/reporting/server/routes/generate_from_savedobject_immediate.ts

```diff
@@ -66,27 +69,22 @@ export function registerGenerateCsvFromSavedObjectImmediate(
     },
     userHandler(async (user, context, req: CsvFromSavedObjectRequest, res) => {
       const logger = parentLogger.clone(['savedobject-csv']);
-      const jobParams = getJobParamsFromRequest(req, { isImmediate: true });
+      const jobParams = getJobParamsFromRequest(req);
       const createJob = createJobFnFactory(reporting, logger);
       const runTaskFn = runTaskFnFactory(reporting, logger);
 
       try {
         // FIXME: no create job for immediate download
-        const jobDocPayload = await createJob(jobParams, req.headers, context, req);
+        const payload = await createJob(jobParams, context, req);
```

Comment:
This FIXME is related to https://github.com/elastic/kibana/pull/76566/files#r491214606